.travis.yml

```diff
@@ -10,18 +10,6 @@ jdk:
 before_install: gem install bundler -v '< 2' --no-document
 matrix:
   include:
-    - stage: r10k tests
-      rvm: 2.7.0
-    - stage: r10k tests
-      rvm: 2.6.5
-    - stage: r10k tests
-      rvm: 2.5.0
-    - stage: r10k tests
-      rvm: 2.4.0
-    - stage: r10k tests
-      rvm: 2.3.0
```

Comment:
oh right.. we had talked about ideally dropping ruby 2.3.0, but I think we decided that we needed to do that at a major version boundary. We didn't add ruby 2.3.0 to the Github Actions test matrix: https://github.com/puppetlabs/r10k/blob/main/.github/workflows/rspec_tests.yml#L12 Should we?

Reply:
Yeah probably, I'll see about adding it.